website/pages/en/developing/supported-networks.mdx

```diff
@@ -9,19 +9,15 @@ export const getStaticProps = getStaticPropsForSupportedNetworks(__filename)
 
 <SupportedNetworksTable networks={props.networks} />
 
-\* Preliminary network support via the [upgrade Indexer](https://thegraph.com/blog/upgrade-indexer/)  
-&#8288;<sup>†</sup> Supports Substreams
+\* Baseline network support provided by the [upgrade Indexer](https://thegraph.com/blog/upgrade-indexer/).  
+\*\* Integration with Graph Node: `evm`, `near`, `cosmos`, `osmosis` and `ar` have native handler and type support in Graph Node. Chains which are Firehose- and Substreams-compatible can leverage the generalised [Substreams-powered subgraph](/cookbook/substreams-powered-subgraphs) integration (this includes `evm` and `near` networks). &#8288;<sup>†</sup> Supports deployment of [Substreams-powered subgraphs](/cookbook/substreams-powered-subgraphs).
 
-The hosted service relies on the stability and reliability of the underlying technologies, namely the provided JSON RPC endpoints.
+- The hosted service and Subgraph Studio rely on the stability and reliability of the underlying technologies, for example JSON-RPC, Firehose and Substreams endpoints.
+- Subgraphs indexing Gnosis Chain can now be deployed with the `gnosis` network identifier. `xdai` is still supported for existing hosted service subgraphs.
```

Comment:
```suggestion
- Subgraphs indexing Gnosis Chain can now be deployed with the `gnosis` network identifier. `xdai` is still supported for existing hosted service subgraphs.
- If a subgraph was published via the CLI and picked up by an Indexer, it could technically be queried even without support. However, please note that we're currently working on improving integration for Indexers and consumers across the board.  
```

Reply:
Added suggested info